src/plugins/discover/public/shared/index.ts

```diff
@@ -12,3 +12,5 @@
 export async function loadSharingDataHelpers() {
   return await import('../application/helpers/get_sharing_data');
 }
+
+export { ISharingData } from '../application/helpers/get_sharing_data';
```

Comment:
just a quick look, it's late and my local Kibana refuses to bootstrap. could you check the build stats of this PR once it's finished, to be sure there is no significant change because of this? I can review it tomorrow for sure

Reply:
Sure -- nothing actually imports this interface, but Typescript complained without the export

Reply:
Before fc16f85c489
  ![image](https://user-images.githubusercontent.com/908371/114467613-07544280-9b9f-11eb-9c90-db2286432f15.png)

After fc16f85c489:
  ![image](https://user-images.githubusercontent.com/908371/114470737-3f11b900-9ba4-11eb-9d50-699e1953ff7d.png)

It looks like this export actually did not affect the bundle sizes! Still, I'm not sure why this change removed 2 async bundles from `discover` and added an extra 25K to the discover page load bundle.

Reply:
@kertal - I noticed that this PR added an extra 25K to the `discover` page load bundle, which is unexpected. I added fc16f85c489 and watching for how the bundle sizes changes in CI metrics.

Reply:
fc16f85 brought the `discover` page load bundle size back down. Thanks for the tip @kertal!

Reply:
even better, this PR saves a few bytes now!